components/ws-manager/pkg/manager/metrics.go

```diff
@@ -111,7 +111,7 @@ func newMetrics(m *Manager) *metrics {
 			Name:      "workspace_stops_total",
 			Help:      "total number of workspaces stopped",
 		}, []string{"reason", "type", "class"}),
-		totalBackupSuccessCounterVec: prometheus.NewCounterVec(prometheus.CounterOpts{
+		totalBackupCounterVec: prometheus.NewCounterVec(prometheus.CounterOpts{
 			Namespace: metricsNamespace,
 			Subsystem: metricsWorkspaceSubsystem,
 			Name:      "workspace_backups_success_total",
```

Comment:
should the name of the metric be updated as well then?

Reply:
very good catch!